Client/mods/deathmatch/logic/CClientRenderElementManager.cpp

```diff
@@ -242,6 +242,31 @@ CClientWebBrowser* CClientRenderElementManager::CreateWebBrowser(uint uiSizeX, u
     return pWebBrowserElement;
 }
 
+////////////////////////////////////////////////////////////////
+//
+// CClientRenderElementManager::CreateVectorGraphic
+//
+//
+//
+////////////////////////////////////////////////////////////////
+CClientVectorGraphic* CClientRenderElementManager::CreateVectorGraphic(uint width, uint height)
+{
+    // Create the item
+    CVectorGraphicItem* pVectorGraphicItem = m_pRenderItemManager->CreateVectorGraphic(width, height);
+
+    // Check create worked
+    if (!pVectorGraphicItem)
+        return NULL;
+
+    // Create the element
+    CClientVectorGraphic* pVectorGraphicElement = new CClientVectorGraphic(m_pClientManager, INVALID_ELEMENT_ID, pVectorGraphicItem);
```

Comment:
As discussed in Discord, this should probably be done in a separate PR as it would require changes to existing code, shared with quite a few other classes.